test/test_person.py

```diff
@@ -0,0 +1,32 @@
+# test for create_openminds_person function in the main
+import pytest
+from bids2openminds.main import create_openminds_person
+import openminds.latest.core as omcore
+
+# Test data: (full_name, given_name, family_name)
+example_names = [("John Ronald Reuel Tolkien", "John Ronald Reuel", "Tolkien"),
+                 ("Bilbo Baggins", "Bilbo", "Baggins"), ("Xue, G.", "G.", "Xue"),
```

Comment:
new line missing (go with one line per name here for better overview)

Reply:
Done :)